site/docs/windows.md

```diff
@@ -140,17 +140,19 @@ To build C++ targets, you need:
         [known issue](https://github.com/bazelbuild/bazel/issues/3949),
         please upgrade your VS 2017 to the latest version.
 
-*   The `BAZEL_VS` or `BAZEL_VC` environment variable. (They are *not* the same!)
+*   The `BAZEL_VS` , `BAZEL_VC` and `BAZEL_VC_FULL_VERSION` environment variable. (They are *not* the same!)
 
     Bazel tries to locate the C++ compiler the first time you build any
-    target. To tell Bazel where the compiler is, you can set one of the
+    target. To tell Bazel where the compiler is, we provide the
     following environment variables:
 
     *   `BAZEL_VS` storing the Visual Studio installation directory
 
     *   `BAZEL_VC` storing the Visual C++ Build Tools installation directory
 
-    Setting one of these variables is enough. For example:
+    *   `BAZEL_VC_FULL_VERSION` Only for Visual Studio 2017 & 2019, the full version number of your Visual C++ Build Tools
```

Comment:
Does this envvar make sense when VC Community/Enterprise/Professional are installed, or only when BuildTools are?

Reply:
Yes, in 2017 &  2019, the path is like `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools|Community|Enterprise|Professional\VC\Tools\MSVC\<BAZEL_VC_FULL_VERSION>`